entitlement/int_entitlement_test.go

```diff
@@ -0,0 +1,55 @@
+package entitlement
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/docker/libentitlement/secprofile"
+)
+
+func testIntEntitlementEnforce(profile secprofile.Profile, value int64) (secprofile.Profile, error) {
+	if value != 1 {
+		return nil, fmt.Errorf("Unexpected value passed to callback")
+	}
+	return profile, nil
+}
+
+func testIntEntitlementEnforceError(profile secprofile.Profile, value int64) (secprofile.Profile, error) {
+	return nil, fmt.Errorf("An error occurred in the callback")
+}
+
+func TestIntEntitlementNoCallback(t *testing.T) {
+	tests := map[Entitlement]*Result{
+		NewIntEntitlement("", nil): nil,
+		NewIntEntitlement("foo.x=1", nil): {
```

Comment:
WDYT of having the string defined at the top of the function with
```go
value := 1
domain := "foo"
identifier := "x"
intEntStr := fmt.Sprintf("%s.%s=%d", domain, identifier, value)
```

and use those values in the fields and in `testIntEntitlementEnforce` so that we don't have a `if value != 1` test (but instead a `value != refValue` ?).

Reply:
It's a nit but I'd prefer if we could avoid having people adding more tests but with the wrong value and not understanding why it fails. (worst-case we can add a comment for it instead)

Reply:
Yep, good idea, updating!